L103-python-add-typing-to-sync-api.md

```diff
@@ -0,0 +1,79 @@
+Add Typing to gRPC Python Sync API
+----
+* Author(s): XuanWnag-Amos
+* Approver: gnossen
+* Status: Draft
+* Implemented in: Python
+* Last updated: 11/09/2022
+* Discussion at: <google group thread> (filled after thread exists)
```

Comment:
Reminder to create the Google group thread once the gRFC is in a presentable state.